.templates/yeet/Makefile

```diff
@@ -61,6 +61,9 @@ clean:
 	rm -rf $(TARGET).yeet
 	rm -rf $(TARGET)
 
+include/vmlinux.h: $(if $(filter yes, $(CONTAINER)), /sys/kernel/btf/vmlinux, )
+	make vmlinux
+
 vmlinux: $(if $(filter yes, $(CONTAINER)), container, )
 ifeq ($(CONTAINER), yes)
 	docker run --rm -it -v $(CURDIR):/opt/$(TARGET) -w /opt/$(TARGET) $(TARGET) make vmlinux
```

Comment:
can we sudo the bpftool so it works on debian. but get the result to be as the same user?